File: demo/app1/src/main/java/com/serious/demo/impl/TestRestServiceImpl.java
package com.serious.demo.impl;
/*
 * @COPYRIGHT (C) 2023 Andreas Ernst
 *
 * All rights reserved
 */

import com.serious.demo.TestRestService;
import org.springframework.web.bind.annotation.RestController;

/**
 * @author Andreas Ernst
 */
@RestController
class TestRestServiceImpl implements TestRestService {
    public String hello() {
        return "hello";
    }
}

File: demo/app1/src/main/java/com/serious/demo/impl/TestRestServiceImpl.kt
package com.serious.demo.impl

import com.serious.demo.TestRestService
import org.springframework.web.bind.annotation.RestController

/*
* @COPYRIGHT (C) 2023 Andreas Ernst
*
* All rights reserved
*/ /**
 * @author Andreas Ernst
 */
@RestController
internal class TestRestServiceImpl : TestRestService {
    override fun hello(): String {
        return "hello"
    }
}
